<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>numPlayersDialog</class>
 <widget class="QDialog" name="numPlayersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="fourBtn">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>231</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="threeBtn">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>151</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="twoBtn">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>71</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Number of Players:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
